itwin-platform-access/imodels-access-backend/src/BackendIModelsAccess.ts

```diff
@@ -265,15 +304,21 @@ export class BackendIModelsAccess implements BackendHubAccess {
       checkpoint = await this._iModelsClient.checkpoints.getSingle(getSingleCheckpointParams);
```

Comment:
Could this call potentially return a checkpoint which isn't successful? Meaning a failed v2 checkpoint which would probably have no containerAccessInfo? I don't think thats a problem the way the code is setup, since we look for a preceding v2 if theres no containerAccessInfo, but just curious.